src/cljs/athens/parser.cljs

```diff
@@ -1,50 +1,75 @@
 (ns athens.parser
   (:require
-    [instaparse.core :as insta]
+    [athens.parse-helper :as parse-helper]
+    [instaparse.core :as insta :refer-macros [defparser]]
```

Comment:
Might be good to move `src/cljs/athens/parser.cljs ` to `src/cljs/athens/parser.cljc`. That way you can could also apply clj tests on the parser

Reply:
Done, though only the `block-parser` code could be moved, not the `transform` or `parse` code. See the last commit message (to be pushed) for details on that.